src/system/voucher.ts

```diff
@@ -108,12 +108,13 @@ const voucherAchvs: Achv[] = [ achvs.CLASSIC_VICTORY ];
           : VoucherType.PREMIUM;
         const key = TrainerType[trainerType];
         const trainerName = trainerConfigs[trainerType].name;
+        const trainer = trainerConfigs[trainerType];
+        const title = trainer.title === "" ? "" : ` (${trainer.title})`;
```

Comment:
this line makes me upsetti

Reply:
```suggestion
       const title = trainer.title ? ` (${trainer.title})` : "";
```

@Tempo-anon Sorry this is obiously better